routes/comment_routes.js

```diff
@@ -83,7 +83,7 @@ router.post(
     const comment = new Comment();
 
     comment.parent = req.params.parent.substring(0, 500);
-    comment.parentType = req.params.type.substring(0, 500);
+    comment.parentType = req.params.type;
```

Comment:
we should still substring this - as we don't want to create an opportunity for a 10GB database query, for example

Reply:
We really don't need to though. If you look 10 lines higher, we check the `type` against an array of values, the largest of which is 7 characters long, and if it matches none of them, we return a 400 and the code never gets to this point.